src/Native/Runtime/amd64/AsmOffsetsCpu.h

```diff
@@ -8,32 +8,49 @@
 //
 // NOTE: the offsets MUST be in hex notation WITHOUT the 0x prefix
 
+#ifdef UNIX_AMD64_ABI
```

Comment:
with the amount of #ifdefing in here, does it make any sense to create another variant of the file here, whether it be a full amd64_unix subdirectory sibling of 'runtime\amd64' or as a windows/unix split within 'runtime\amd64'?